index.html

```diff
@@ -46,11 +47,31 @@
         height: 256px;
       }
 
+      #bam2-header {
+        height: 40px;
+        width: 100%;
+        background-color: #2d8fc1;
+        position: fixed;
```

Comment:
Rather than using `position: fixed` and setting `margin-top` on `.read-depth-container` above, you can leave position as its default setting and let the header take up space. This should simplify things. You also won't need z-index, top, or left.

Reply:
Are we going to align with the prod version in bam1? In the bam1, the banner is placed in fixed position and the top and left are set up to 0;

Reply:
Seems overcomplicated to me. Unless it won't work for some reason I vote for the simpler way.